bitnami/elasticsearch/templates/curator/cronjob.yaml

```diff
@@ -91,7 +89,7 @@ spec:
                 - name: config-volume
                   mountPath: /etc/es-curator
                 {{- if .Values.curator.extraVolumeMounts }}
-                {{- toYaml .Values.curator.extraVolumeMounts | nindent 16 }}
+                {{- include "common.tplvalues.render" (dict "value" .Values.curator.extraVolumeMounts "context" $) | nindent 16 }}
                 {{- end }}
               {{- if .Values.diagnosticMode.enabled }}
```

Comment:
I don't think it makes sense to run a cronjob with a sleep command in the case of using diagnosticMode